samples/drivers/adc/adc_dt/boards/beagleconnect_freedom.overlay

```diff
@@ -0,0 +1,73 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2024 Ayush Singh <ayush@beagleboard.org>
+ */
+
+/ {
+	zephyr,user {
+		/* Reading mulitple pins not possible */
+		io-channels = <&adc0 0x09>;
+	};
+};
+
+&adc0 {
+	status = "okay";
+
+	#address-cells = <1>;
+	#size-cells = <0>;
+
+	/* MB1 AN */
+	channel@9 {
```

Comment:
Is it possible to add "label" rather than just putting "MB1 AN" in the comments?

Reply:
I have added the label.